integration-tests/src/test/scala/tech/beshu/ror/integration/plugin/FieldRuleMSearchApiPluginTests.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.integration.plugin
+
+import tech.beshu.ror.integration.suites.fields.FieldRuleMSearchApiSuite
+import tech.beshu.ror.integration.utils.SingletonPluginTestSupport
+
+class FieldRuleMSearchApiPluginTests extends FieldRuleMSearchApiSuite with SingletonPluginTestSupport
```

Comment:
I'd rather have one suite with all FLS tests. New test approach assumes that integration tests will reflect ES API (eg. snapshots api, index api) or concepts (like FLS, DLS).